src/kernel/include/shared/consts.h

```diff
@@ -44,20 +44,25 @@ struct pt_regs {
 #endif
 #define PAGE_ORDER 12
 #ifndef __KERNEL__
-#define PAGE_SIZE (1<<PAGE_ORDER)
+#define PAGE_SIZE (1 << PAGE_ORDER)
 #endif
 
 #define MAX_SERVICE_DEPTH 31
 #define MAX_NUM_THREADS 64
+
 /* Stacks are 2 * page_size (expressed in words) */
```

Comment:
+1 @base0x10 

Comments fall out of coherence with code easily.  Removing them if this is likely to happen is often the right move.